_pytest/config.py

```diff
@@ -399,13 +399,15 @@ def consider_conftest(self, conftestmodule):
             self.consider_module(conftestmodule)
 
     def consider_env(self):
-        self._import_plugin_specs(os.environ.get("PYTEST_PLUGINS"))
+        specs = os.environ.get("PYTEST_PLUGINS")
+        if specs:
+            plugins = specs.split(',')
```

Comment:
thanks for taking a deeper look, the location you mention is indeed a better place

my feat of the behaviour change has 2 parts 1. we now pass more different types trough the functions (before it was None or a string, now it is None, a string or a list)
upon investigation of course i noticed that the `_import_plugin_specs` does filter "false" values
we may experience misbehavior on the value "," however (which would also hold prior to the change)